src/Microsoft.AspNet.Mvc.Core/MvcOptions.cs

```diff
@@ -138,5 +139,12 @@ public int MaxModelValidationErrors
         /// when it contains the media type */*. <see langword="false"/> by default.
         /// </summary>
         public bool RespectBrowserAcceptHeader { get; set; } = false;
+        
+        /// <summary>
+        /// Gets a Dictionary of CacheProfile Names, <see cref="CacheProfile"/> which are pre-defined settings for
+        /// <see cref="ResponseCacheFilter"/>.
+        /// </summary>
+        public Dictionary<string, CacheProfile> CacheProfiles { get; }
+            = new Dictionary<string, CacheProfile>(StringComparer.OrdinalIgnoreCase);
```

Comment:
why ignore case? I generally find it surprising when a programmatic identifier is case-insensitive.